src/SFA.DAS.ApplyService.InternalApi/Services/AssessorLookupService.cs

```diff
@@ -0,0 +1,128 @@
+﻿using SFA.DAS.ApplyService.Application.Apply.Roatp;
+
+namespace SFA.DAS.ApplyService.InternalApi.Services
+{
+    public class AssessorLookupService : IAssessorLookupService
+    {
+        public string GetTitleForSequence(int sequenceId)
+        {
+            switch (sequenceId)
+            {
+                case RoatpWorkflowSequenceIds.ProtectingYourApprentices:
+                    return "Protecting your apprentices checks";
+                case RoatpWorkflowSequenceIds.ReadinessToEngage:
+                    return "Readiness to engage checks";
+                case RoatpWorkflowSequenceIds.PlanningApprenticeshipTraining:
+                    return "Planning apprenticeship training checks";
+                case RoatpWorkflowSequenceIds.DeliveringApprenticeshipTraining:
+                    return "Delivering apprenticeship training checks";
+                case RoatpWorkflowSequenceIds.EvaluatingApprenticeshipTraining:
+                    return "Evaluating apprenticeship training checks";
+                default:
+                    return null;
+            }
+        }
+
+        public string GetTitleForPage(string pageId)
+        {
+            switch (pageId)
```

Comment:
Personally, I'd prefer to put this into an appsettings section rather than a case statement

Reply:
Makes sense if these titles are susceptible to change but they are static for now